tools/hangar/__snapshots__/test_corpus/valid/for_loop.w.ts.snap

```diff
@@ -87,6 +87,119 @@ constructor() {
     {((cond) => {if (!cond) throw new Error(\`assertion failed: '(post_continue_hits === 0)'\`)})((post_continue_hits === 0))};
   }
   {((cond) => {if (!cond) throw new Error(\`assertion failed: '(j === 3)'\`)})((j === 3))};
+  {console.log(\\"---\\\\nfor x in 0..0 { ... }\\")};
+  for (const x of ((s,e,i) => { function* iterator(start,end,inclusive) { let i = start; let limit = inclusive ? ((end < start) ? end - 1 : end + 1) : end; while (i < limit) yield i++; while (i > limit) yield i--; }; return iterator(s,e,i); })(0,0,false)) {
```

Comment:
I expected this to be a call to an `std` function instead of inline.

Something like:

```suggestion
  for (const x of std.range(s,e,i)) {
```

Reply:
In the preflight, the Wing library is imported, so it's possible to generate the loop with `std.Range` when compiling, but in the inflight, the lambda doesn't have the Wing SDK imported, so I had to leave the function inline.

I added a loop inside a function to generate a snapshot in the existing test.

Reply:
That's a good point about preflight versus inflight.

Kind of feels like we need to import at least a portion of our stdlib into inflight but that's not for now. Can you open an issue so we can track?

Ok let's keep this as is at this point.